---
steps: 3000
time_step: 0.1
save_directory: ''
record_pose: true
runs: 1
scenario:
  type: Antipodal
  position_noise: 0.005
  orientation_noise: 0.01
  radius: 2
  target_margin: 0.1
  tolerance: 0.2
  groups:
    -
      type: thymio
      number: 20
      radius: 0.1
      control_period: 0.1
      speed_tolerance: 0.01
      kinematics:
        type: 2WDiff
        wheel_axis: 0.094
        max_speed: 0.166
      behavior:
        type: HL
        optimal_speed: 0.08
        horizon: 10.0
        safety_margin: 0.2
      state_estimation:
        type: Bounded
        range_of_view: 10.0
